Content/OpenClipboard.md

```diff
@@ -0,0 +1,13 @@
+# Open Clipboard
+
+_OpenClipboard_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v0.7.5 / 5.49.8. It allows for clipboard access for a sandbox to be disabled. For example:
```

Comment:
```suggestion
_OpenClipboard_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v0.7.5 / 5.49.8. It allows to disable clipboard access for a sandbox. For example:
```